lib/getPackageReportData.js

```diff
@@ -53,9 +53,18 @@ function getPackageReportData(packageEntry, callback) {
 
 		var author = json.author
 		if (_.isObject(author)) {
-			author = author.name;
-			if (author.email) { author = author + ' ' + author.email; }
-			if (author.url) { author = author + ' ' + author.url; }
+			author = author.name || ""
+
+			if (author.email) {
+			  if (author.length > 0) {
+			    author += ' ';
+			  }
+			  author += author.email;
+			}
+
+			if (author.url) {
+			  author += "" + author.url;
```

Comment:
1. please use single ticks for strings to be consistent
2. if author.name and / or author.email exist, then we need a space character. If neither exist then author = author,url.

So I think this line should look like line 59...62 or did I miss the point?

Reply:
Make sense.